ReactNativeClient/lib/components/screens/config.js

```diff
@@ -391,7 +391,7 @@ class ConfigScreenComponent extends BaseScreenComponent {
 						<Text key="label" style={this.styles().switchSettingText}>
 							{md.label()}
 						</Text>
-						<Switch key="control" style={this.styles().switchSettingControl} value={value} onValueChange={value => updateSettingValue(key, value)} />
+						<Switch key="control" style={this.styles().switchSettingControl} trackColor={{ false: '#969696' }} value={value} onValueChange={value => updateSettingValue(key, value)} />
```

Comment:
Please change these to values from the theme. Maybe this color can be integrated with style? If not it should still draw from the theme color.

Reply:
Sir, I tried it but it seems that it could not be integrated with style.But yeah, i will draw it from the theme color.

Reply:
@CalebJohn I have made the changes.Could you please review it